coxeter/shapes/polyhedron.py

```diff
@@ -356,9 +356,36 @@ def vertices(self):
 
     @property
     def faces(self):
-        """list(:class:`numpy.ndarray`): Get the polyhedron's faces."""
+        """list(:class:`numpy.ndarray`): Get the polyhedron's faces.
+
+        Results returned as vertex index lists.
+        """
         return self._faces
 
+    @property
+    def edges(self):
+        """list(:class:`numpy.ndarray`): Get the polyhedron's edges.
+
+        Results returned as vertex index pairs.
+        """
+        edges = []
+        for face in self.faces:
+            [
+                edges.append((i, j))
+                for i, j in zip(np.append(face, face[0]), np.append(face[1:], face[0]))
```

Comment:
`np.roll` is the nice way to do this!
```suggestion
                for i, j in zip(face, np.roll(face, -1))
```

Reply:
Looks good - I have taken this into account with the changes in the next subconversation.